cmd/serve/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/shraddha1704/goyagi/pkg/server"
+	"github.com/lob/logger-go"
+)
+
+func main() {
+	log := logger.New()
+
+	srv := server.New()
+
+	log.Info("server started")
+
+	err := srv.ListenAndServe()
+	if err != nil && err != http.ErrServerClosed {
+		log.Err(err).Fatal("server stopped")
+	}
+
+	log.Info("server stopped")
+}
```

Comment:
Can you make sure all your files end with a newline character? In VSCode, you can set the `    "files.insertFinalNewline"` setting to `true` to do this automatically.

Reply:
Yes. I have added that in the settings. Will be updating them soon!